mep/accounts/admin.py

```diff
@@ -49,6 +49,13 @@
     ),
 }
 
+
+class OpenFootnoteInline(FootnoteInline):
+    '''Customize footnote inline for borrowing and purchase events.'''
+    classes = ('grp-collapse', )  # grapelli collapsible, but not closed
+    extra = 0
```

Comment:
I set extra to 0 for borrowing events so it would show the one created on import without an extra one, but for data entry on purchases we expect them to be adding footnotes. I think maybe we should set this to 1 instead of 0.